sdk/daml-lf/validation/BUILD.bazel

```diff
@@ -98,10 +123,16 @@ da_scala_test_suite(
 # TODO https://github.com/digital-asset/daml/issues/17254
 #  this test should not use canton and be part of the suite above
```

Comment:
I guess this comment needs to be copied to the test suites you're splitting out?